wiki/Tournaments/JHC/JHC_2024/en.md

```diff
@@ -0,0 +1,213 @@
+    - Exclusion of specific players for one beatmap.
+    - Exclusion of specific players for an entire match.
+    - Declaring the match as forfeited, or as a win by default for the other team.
+    - Disqualification from the entire tournament.
+    - Disqualification from the current and future official tournaments, until appealed.
+18. Referees may allow, at their discretion, lower or higher tolerances for `timers`.
+19. The tournament managers may request liveplays or recordings of individual players or teams at any point in the tournament without prior warning.
+20. Any tournament-related complaints shall be sent to osu! official report form.
+21. The tournament managers reserve the right to modify these rules at any moment. Any such changes will be announced in advance.
+
+### Tournament registration
+
+1. Please fill [Google Form](https://forms.gle/guwZqn8513SuFAYe8) or [Tencent Form](https://docs.qq.com/form/page/DUFlBd2NzYXVqTGxO) to register.
+   - Player **must** join this [Discord server](https://discord.gg/DqPuRU8Bhc) or this [QQ Group](https://qm.qq.com/cgi-bin/qm/qr?_wv=1027&k=pRNiqDnTUmLKQgy8iy2LnLx3uTYIL0dG) after register.
+2. To ensure valid registrations, every participant will be sent to the [tournament screening](/wiki/Tournaments/Official_support#tournament-screening).
+   - Players must not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
+3. A list of all successfully registered players will be published after the registration.
+4. [Tournament staff members](/wiki/Tournaments/Official_support#staff) must not participate as players in the tournament.
+
+### Qualifier instructions
+
+1. In the Qualifier stage, all teams will play a specific pool designed by the mappool selectors.
+2. The Qualifier pool contains 6 maps, all of which will use [Free Mod](/wiki/Gameplay/Game_modifier#freemod) rules. The types of each maps are:
+   - DS | Dense Chordjack
+   - MD | Middle Chordjack
+   - LT | Light Chordjack
+   - RM | Rhythm Jack
+   - HV | Heavy Jack (Anchor Jack & Full Jack)
+   - SP | Speed Jack (Quadstream & Minijack)
+3. Players will play the mappool at least once (and up to twice) at a designated time. Their best combined score will be the final result.
+4. The mappool will be played in the following order: DS, MD, LT, RM, HV, SP. Players may not ask to play the pool in any other order.
+   - Teams may skip playing any beatmap on their second run.
+5. For each map, each team must have **only 1 member** to play. They may be exchanged freely after a map is concluded.
+6. An optional 5-minute break will be offered between the first and second playthrough of the mappool.
+7. All teams will play their qualifiers in separate rooms.
+   - It is *suggested* that teams do not broadcast or share their results publicly to avoid seed manipulation.
+8. The top 32 seeded teams will advance to the Round of 32, and \#33 ~ \#48 seeded teams will advance to the Loser Round 1.
```

Comment:
what is "Loser Round 1"? there is no mention of it in the schedule nor anywhere else